README.md

```diff
@@ -44,13 +44,13 @@ Run and monitor the program:
 Flash the program:
 
 ```bash
-./esp-cargo espflash flash --release --example http
+./esp-cargo espflash flash --release --example led
```

Comment:
Why are we changing the names?

Reply:
Added this to a separate commit to better explain.